packages/react-media/providers/MediaProvider.tsx

```diff
@@ -91,29 +30,31 @@ interface AudioProviderProps {
   children?: ReactNode;
 }
 
-export function AudioProvider({
+export function MediaProvider({
+  element: initialElement,
   tracks: initialTracks = [],
   cursor: initialCursor = 0,
   autoPlay: initialAutoPlay = true,
   repeat: initialRepeat = false,
   shuffle: initialShuffle = false,
   children,
-}: AudioProviderProps) {
-  const [tracks, setTracks] = useState<AudioTrack[]>(initialTracks);
-  const [shuffled, setShuffled] = useState<AudioTrack[]>(
-    initialShuffle ? shuffleArray([...initialTracks]) : initialTracks
+}: MediaProviderProps) {
+  const [tracks, setTracks] = useState<MediaTrack[]>(initialTracks);
+  const [shuffled, setShuffled] = useState<MediaTrack[]>(
+    initialShuffle ? shuffleArray([...initialTracks]) : initialTracks,
   );
   const [cursor, _setCursor] = useState<number>(initialCursor);
 
-  const track = useMemo<AudioTrack | undefined>(
+  const track = useMemo<MediaTrack | undefined>(
     () => shuffled[cursor],
-    [shuffled, cursor]
+    [shuffled, cursor],
   );
   const trackRef = useSynchronizedRef(track);
 
-  const element = useMemo(
-    () => (typeof Audio !== "undefined" ? new Audio() : undefined),
-    []
+  const elementRef: MaybeRef<HTMLMediaElement> =  useRef(
```

Comment:
@thijsdaniels This part of the provider was rendering the hook conditionally and that's why it's the only difference comparing to the section library version.

Reply:
I had a linter error.

Reply:
Let me try to find the exact error.

Reply:
@thijsdaniels this is what you get if you keep it the way it is in the section library.
<img width="1148" alt="image" src="https://github.com/codedazur/toolkit/assets/26919126/3a5029d6-8e68-4210-969d-617dabc4a89e">